README.md

```diff
@@ -44,6 +44,28 @@ but unnamed temporary credentials can be created regardless of your scopes.## AP
 The REST API methods are documented on
 [http://docs.taskcluster.net/](http://docs.taskcluster.net/)
 
+## Sync vs Async
+
+The objects under `taskcluster` (e.g., `taskcluster.Queue`) are python2-compatible and operate synchronously. This allows for taskcluster api access, though for bulk operations will be considerably slower... more than 10x slower in cases where you need to run many api calls that would benefit from concurrency.
+
+The objects under `taskcluster.async` (e.g., `taskcluster.async.Queue`) require `python>=3.5`. `taskcluster.async` will only be available under the python3 wheel or the source distribution; the python2 wheel will not include it. The async objects use asyncio coroutines for concurrency; this allows us to put I/O operations in the background, so operations that require the cpu can happen sooner. The code would look something like
```

Comment:
The "`taskcluster.async` will only be available under the python3 wheel or the source distribution; the python2 wheel will not include it." bit is now no longer the case, since it was going to be a bit of a hack in #82.